docs/_posts/testing_a_tricky_edge_case_in_a_tuf_client.md

```diff
@@ -0,0 +1,104 @@
+
+Now imagine that we have performed an update and stored metadata in a cache. And the locally stored timestamp/snapshot has expired. But we still need it to perform an update from remote by verifying the signatures and we need to use the expired timestamp. Here the repository simulator can do a great job testing this scenario, as we’d like to somehow update expiry without explicitly touching the file system from the tests.
+
+### Example 1:
+We can play with versions and expiry to verify that this scenario not explicitly mentioned in the spec works correctly and safely. By using the simulator, we can do the following:
+* Set the timestamp expiry one week ahead (Day 7)
+* On the very first day download, verify, and load metadata for the [top-level roles](https://theupdateframework.github.io/specification/latest/#roles-and-pki) following the specified order and implementing all the checks required in the TUF client workflow (which we simply call a refresh)
+* Then we bump [snapshot](https://theupdateframework.github.io/specification/latest/#update-snapshot) and [targets](https://theupdateframework.github.io/specification/latest/#targets) versions to v2 on the same day and set v2 expiry dates three weeks ahead (Day 21)
+
+Then if we travel in time somewhere between Day 7 and Day 21, we’d like to perform a successful refresh with the expired locally cached timestamp and we want the final version of the snapshot and targets roles to be v2.
+This is a not so obvious use-case to keep in mind when thinking about updates. You can see how it looks in practice in the [reference implementation](https://github.com/theupdateframework/python-tuf/blob/develop/tests/test_updater_top_level_update.py#:~:text=test_expired_metadata).
+
+### Example 2:
+Now let’s see if a rollback attack protection can be performed when the local timestamp has expired. In this case we need at least two timestamp and snapshot versions, an expired older version of timestamp, and a verification that a rollback check is performed with the old version.
+
+For a timestamp rollback, the case is pretty similar to the use of expired metadata. We can do the following:
+* Set timestamp v1 expiry one week ahead (Day 7)
+* Perform refresh on the very first day
+* Publish timestamp v2 with expiry three weeks ahead (Day 21)
+* Perform a refresh somewhere between Day 7 and Day 21
+
+All we need is to verify that rollback check uses the expired timestamp v1. (For reference, see the implementation [example](https://github.com/theupdateframework/python-tuf/blob/develop/tests/test_updater_top_level_update.py#:~:text=test_expired_timestamp_version_rollback)).
```

Comment:
I think it will be better if you move this sentence to the bullet points.
With the usage of bullet points (or numbers), one will expect that all of the test steps are described, but here they are not. There is one additional step to check which timestamp is used which is mentioned here.